sbi/inference/posteriors/base_posterior.py

```diff
@@ -678,7 +682,8 @@ def sample_conditional(
                 `init_strategy` for the initialisation strategy for chains; `prior`
                 will draw init locations from prior, whereas `sir` will use Sequential-
                 Importance-Resampling using `init_strategy_num_candidates` to find init
-                locations.
+                locations, `enable_transform` a bool indicating whether MCMC is
+                performed in z-scored (and unconstrained) space.
```

Comment:
this gets a bit difficult to follow. maybe starting a new line for each sub argument improves readability?